server/graphql/v1/mutations/paymentMethods.js

```diff
@@ -195,3 +195,32 @@ export async function updatePaymentMethod(args, remoteUser) {
 
   return models.PaymentMethod.update(pick(args, allowedFields), { where: { id: paymentMethod.id } });
 }
+
+/** Update payment method with given args */
+export async function replaceCreditCard(args, remoteUser) {
+  const oldPaymentMethod = await models.PaymentMethod.findByPk(args.id);
+  if (!oldPaymentMethod || !remoteUser || !remoteUser.isAdmin(oldPaymentMethod.CollectiveId)) {
+    throw PaymentMethodPermissionError;
+  }
+
+  const createArgs = { ...omit(args, ['id']), service: 'stripe', type: 'creditcard' };
+
+  const newPaymentMethod = await createPaymentMethod(createArgs, remoteUser);
```

Comment:
We're not checking that `remoteUser` is an admin of `createArgs.CollectiveId`, so in the current state users can use this endpoint to create payment methods for collectives they don't own.

Reply:
@Betree can you double check? `createPaymentMethod` should be taking care of that

Reply:
Ah yes didn't see that. All good then!